code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
most watchlists that I see are constant greytide, which they should know by now after the several previous warnings; or metagame watch.

The note should default to secret, the admin can just click the note in PP for the niche issue that they should be informed they're on the watchlist.